modules/pull.lua

```diff
@@ -854,17 +854,20 @@ function Module:CheckGroupForPull(classes, resourceStartPct, resourceStopPct, ca
 
         if member and member.ID() > 0 then
             if not classes or classes:contains(member.Class.ShortName()) then
-                local resourcePct = self.TempSettings.PullState == PullStates.PULL_GROUPWATCH_WAIT and resourceStopPct or resourceStartPct
+                local resourcePct = self.TempSettings.PullState == PullStates.PULL_GROUPWATCH_WAIT and resourceStartPct or resourceStopPct

```

Comment:
The way this reads is 

Loop 1:
Idle == Wait and Start or Stop -> Stop
Wait == Wait and Start or Stop -> Start

Are you use you don't have any other local changes because this new logic looks reversed to me.

Reply:
according to the original logic though it was doing 
loop 1 
idle != Wait and stop or start -> start
Loops 2 
Wait == Wait and Stop or Start -> stop. 

which was backward.

Reply:
![image](https://github.com/DerpleMQ2/rgmercs/assets/124466615/e33f5936-e9cd-4955-9273-d610b4b20387)

Reply:
You are right.
Stop means Stop Pulls at 40%
Start means RESUME at 80%

Let's rename them to Stop and Resume I think it should make it clearer.